src/swapd/runtime.rs

```diff
@@ -612,46 +552,42 @@ impl Runtime {
                         )?;
                     }
                 }
-                // forward the message to wallet to trigger creation of reveal message
-                debug!("{} | forward maker commitment to wallet", self.swap_id);
-                self.send_wallet(
```

Comment:
As `send_wallet` is now unused we can remove it from `service.rs`, I linked the related issue already.

Reply:
[f3dfc10](https://github.com/farcaster-project/farcaster-node/pull/789/commits/f3dfc105803216777d7857558f05a7ae94c9f439)